packages/x-date-pickers/src/internals/models/value.ts

```diff
@@ -0,0 +1,5 @@
+import { PickerValidDate } from '../../models/pickers';
+
+export type PickerValue = PickerValidDate | null;
+
+export type PickerRangeValue = [PickerValidDate | null, PickerValidDate | null];
```

Comment:
Nipick: Is there a need to export it from the community package? 🤔 
Or do you think that it looks cleaner exporting similar model types from the same place?

I'm not strictly against it, just curious about the arguments for this choice. 😉

Reply:
For now it's not exported at all.
It needs to be defined here so that we can replace `TValue` with `TIsRange` in a follow up PR. That way we can infer the value from `TIsRange` in all the interfaces defined in the community package without having the pro package passing it as a generic.
Should it be exported? The more I think about it, the more I'm in favor of keeping `DateRange` (with a better name) from the pro package with a generic and use it purely as a utility type to help typing the `useState` when using a range component. And then use the purely internal `PickerRangeValue` for our typing.

WDYT?

Reply:
Gotcha. The direction makes sense. Let's go with it and course-correct if needed. 👍